### PR TITLE
User Error Addition

### PR DESCRIPTION
**Pull request by Xmetalfanx**
(new to  git pull requests so I am not sure if I am doing this right) 

I haven't gotten this 100% yet, and I admit I have not touched the "**language**" numbers ... didn't know if i should, so I left it ... 

Right now in a test on LibreElec 7.9 alpha on my pi2 (alpha version based on Kodi 17) and my install of Kodi 15.2 (??) I think it is on Ubuntu Mate on my laptop it, this sort of works ... it displays the user error graphic that I downloaded from the JB site and it plays the stream in  video mode

Issue: The one issue I see so far is that its not showing "User error" in the Kodi Menu .. it's showing a blank option and when you click that ,  that's when the user error show-list comes up

### CODE REVIEW

**Comment by RobLoach** [comment]
Looking great so far! Nicely done, mind adding the Language entries in here?

**Comment by Xmetalfanx** [comment]
I will look over the code, but I have to be honest, I am not sure how that'd be done... I did add a number to the  **language**  number of the show with the highest number there already, .... so it wouldn't (have no idea if it would) conflict.

 I admit right off, that I am not sure what I am doing with the Language entry

**Comment by Xmetalfanx** [comment]
Ok I figured it out ... the Show title is Showing now and I added the "Description" too in the Language file.

seems to work fine now with my tests

**Comment by Xmetalfanx** [comment]
I have been tweaking a few things, but the main issue I am seeing is in the episode list menu, it's not showing the user error graphic for the individual episodes

**Comment by rickyphewitt** [comment]
@Xmetalfanx,

I'll take a look. Just getting my feet wet with kodi plugin development so no promises :)

**Comment by rickyphewitt** [comment]
Taking a look at the XML feeds, the User Error is different than the rest of the JB shows (just compared the xml output after its parsed by soup on unfilter). Looking at Unfilter's feed: within each <item>  there is a  <media:thumbnail url="http://somethumnailimagehere.png" />. The code looks for the media:thumbnail and adds that as the per episode images. There is no media:thumbnail tag (or other thumbnail tag that stands out to me) in the UserError feed. 

My suggestion is to use the default Show image until a proper thumbnail is added in the feed. Thoughts?

**Comment by rikai** [comment]
We are testing a different method to publish user error than our other shows, so this unlikely to change unless the method changes, so a default thumbnail probably makes sense.

There's an image defined at the top of the RSS  feed that should work fine for this use case.

**Comment by rickyphewitt** [comment]
I was able to make a few changes to @Xmetalfanx code and add default images for any shows episodes that do not have an episode thumbnail. However I'm not sure how to merge it as its based off of @Xmetalfanx code. 

Here is a link to the commit I added: https://github.com/rickyphewitt/plugin.video.jupiterbroadcasting/commit/edfe77f4a5fc08e2b00aecfd4aaebb5ecb0a7737

@Xmetalfanx feel free to incorporate it into your commit if it fits the bill.

Images are Below. 1 shows User error episode browser, the other shows Unfilter with perbased episode thumbnails is not affected.
unfilterwithperepisodethumnail
usererrorepisodeswithimage

**Comment by rikai** [comment]
I believe the way to do that would be to submit a PR to [his branch that this PR is based off of](https://github.com/Xmetalfanx/plugin.video.jupiterbroadcasting/tree/frodo).

**Comment by rickyphewitt** [comment]
Done

**Comment by Xmetalfanx** [comment]
Honestly I am probably newer at this ( git, kodi addons, and some of this code) than any of you lol ... I just took a look at the code and copy/pasted/changed a thing or two to add User Error.  I THINK I merged the changes (if not, I intended to)/

Really I just dont want to get too many forks of forks of forks of the addon, I just want to do what I can to help JB out.  

Thank you for making the changes by the way ... I was going to do that when I had a chance today (just now) but it appears it's already  been done.

**Comment by Xmetalfanx** [comment]
Also i have not 100% seen if this was an issue but playing with the addon on my Pi2 with Libre Elec 7.9xxx.... alpha (Kodi 17 based) ... I noticed that only some of shows were playing here 

Episode 5, Episode 4  ... some of the others didn't play but then again I have Comcast here so that may have been something on my end.

**Comment by Xmetalfanx** [comment]
oops sorry about that close/ reopen thing ... i will just leave those buttons alone lol ... I thought i was on the page for "my" fork when i hit closed.

**Comment by rikai** [comment]
I think the first few episodes may have some difficulty playing on certain video players because we were using different encoding settings from what we normally do, I may go into the back archive and update them to the new encoding when they have a chance but I wouldn't worry about it too much in the meantime

**Comment by rikai** [comment]
@Xmetalfanx My only other suggestion for this PR is to maybe [use the image from the other PR](https://github.com/RobLoach/plugin.video.jupiterbroadcasting/blob/usererror/resources/media/usererror.png) rather than the one you've currently got committed. Better dimensions and quality. 👍

**Comment by Xmetalfanx** [comment]
No problem @ Rikai ... just wanted to make sure I didn't accidentally change something that messed things up  (@ the playback/ encoding thing)

**Comment by Xmetalfanx** [comment]
sorry guys ... (Thanks Rikai. btw) for the links ... I understand what I am reading but when I try to put what I see into practice with squash, things seem to get duplicated or commits seem to be added, instead of combined ... so to speak.

I am not touching my local copy of the fork, but rather I am playing with a different project I am working on  (after all if i bork that up, I know how to redo things to get it to the latest point) 

This is where I am more of a "I need to try it myself in the 'real world' type of visual learner" ... sorry .. please bear with me here.  I want to do a good job cleaning extra commits up, but I think my brain is fried today, lol

File: addon.xml [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
-    version="3.2.1"
+<<<<<<< HEAD
+<<<<<<< HEAD
+    version="3.2.4"
+=======
+    version="3.2.3"
+>>>>>>> dd2fdd6... 3.2.3
+=======
+    version="3.2.4"
+>>>>>>> 16eb1eb... Update BeautifulSoup to 3.2.1
```
Comment (RobLoach) [inline addon.xml:13]: Some merge artifacts.

**Pull request merged**